mojo/core/broker_castanets.cc

```diff
@@ -262,6 +251,13 @@ void BrokerCastanets::OnBufferSync(uint64_t guid_high, uint64_t guid_low,
   memcpy(ptr + offset, data, sync_bytes);
   munmap(ptr, sync_bytes + offset);
 
+  BufferSyncAckData* sync_ack;
+  Channel::MessagePtr out_message = CreateBrokerMessage(
+      BrokerMessageType::BUFFER_SYNC_ACK, 0, 0, &sync_ack);
+  sync_ack->guid_high = guid_high;
+  sync_ack->guid_low = guid_low;
+  channel_->Write(std::move(out_message));
+
```

Comment:
Is it possible that BUFFER_SYNC_ACK gets skipped by line 234 even if a BUFFER_SYNC_ACK msg must be sent?